scripts/claude-slide/verify_slide_mechanical.py

```
@@ -0,0 +1,823 @@
+def fold_cosmetic(s: str) -> str:
+    """表記ゆれ判定用の正規化。これで一致するペアは「改変」でなく「表記ゆれ」扱い。
+    - NFKC で全角→半角（＆→&、（→(、：→:、全角数字→半角 等）
+    - 波ダッシュ系を ~ に統一（NFKC で畳まれない 〜 U+301C を明示的に処理）
+    - すべての空白を除去（CJK 間の余分なスペース差・改行差を無視）
+    内容（可読文字）の追加・削除・置換は畳まれないので、真の改変は検出され続ける。"""
+    s = unicodedata.normalize("NFKC", s)
+    s = s.replace("〜", "~").replace("～", "~").replace("∼", "~")
+    # 箇条書き記号の差（元 Doc の「・」↔ 生成の「-」「‐」「•」等）は内容ではなく表記。
+    # src/gen 双方に対称適用するので、ダッシュだけ違う文以外を誤って一致させることはない。
+    # カタカナ長音符「ー」は実単語の一部なので除外する。
+    s = re.sub(r"[・•·‐―−\-]", "", s)
```

Ruff (0.15.20)

[warning] String contains ambiguous `‐` (HYPHEN). Did you mean `-` (HYPHEN-MINUS)?

(RUF001)